_posts/2020-11-25-flyway-7.3.0.md

```diff
@@ -0,0 +1,65 @@
+---
+layout: blog
+subtitle: Flyway 7.3.0 Released
+permalink: /blog/flyway-7.3.0.html
+author: julia
+---
+
+**Flyway 7.3.0** is out! This release contains new features and improvements over Flyway 7.2.0.
+
+## Highlights
+- A placeholder for the full path of a migration file
```

Comment:
The placeholder doesn't provide the full path, just the full filename, as opposed to just the description (`V1__test.sql` instead of `test`)